src/Slawek/Fsd.Slawek.Cs.Ex2/Fsd.Slawek.Cs.Ex2/Models/AnimalPresenter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Fsd.Slawek.Cs.Ex2.Models
+{
+    public class AnimalPresenter
+    {
+        public void IntroduceAnimal(Animal animal)
+        {
+            GetAnimalName(animal);
+
+            var myVoice = animal as IAnimalVoice;
```

Comment:
nie musisz rzutować na interfejs. Jeżeli klasa implementuje dany interfejs to kompilator sobie to spokojnie ogarnie. ta linijka jest zbędna